main.rb

```diff
@@ -0,0 +1,139 @@
+require_relative 'planet'
+require_relative 'solar_system'
+
+
+#
+# SolarSystem = [mars, venus, jupiter, moon, earth, pluto]
+
+def main
+
+  puts "Would you like to play the planet info game?"
+  answer = gets.chomp
+
+#instance of solarsystem
+  solar_system = SolarSystem.new('Sol')
+
+#instance of planet
+  earth = Planet.new('earth', 'blue-green', '444', '10.54', 'humans live on it')
+  mars = Planet.new('mars', 'red', '666', '1092.3', 'its red')
+  venus = Planet.new('venus', 'sliver', '3030', '99922', 'its super far away')
+
+  solar_system.add_planets(earth)
```

Comment:
This line reads a little weirdly to me... `solar_system.add_planets(earth)`, and we give it one instance of one `Planet`. Maybe it would make more sense to call this method `add_planet` (singular)?